src/Compilers/CSharp/Portable/Operations/CSharpOperationFactory.cs

```diff
@@ -42,8 +42,13 @@ public IOperation Create(BoundNode boundNode)
             return _cache.GetOrAdd(boundNode, n => CreateInternal(n));
         }
 
-        private IOperation CreateInternal(BoundNode boundNode)
+        private IOperation CreateInternal(BoundNode boundNode, bool createParenthesized = true)
         {
+            if (createParenthesized && boundNode.Syntax.Parent is ParenthesizedExpressionSyntax)
+            {
+                return CreateParenthesizedExpression(boundNode);
```

Comment:
>return CreateParenthesizedExpression(boundNode); [](start = 16, length = 48)

We probably shouldn't do this for lowered tree. Perhaps we should make a note about this somewhere.